package.json

```diff
@@ -6,13 +6,22 @@
   "scripts": {
     "dev": "./node_modules/.bin/webpack-dev-server --mode=development",
     "build": "./node_modules/.bin/webpack --mode=production",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "cd dist && ../node_modules/.bin/http-server -c-1 -p 8081 & open http://localhost:8081/index.html & wait",
+    "lint": "./node_modules/eslint/bin/eslint.js src/",
+    "lint:fix": "./node_modules/eslint/bin/eslint.js src/ --fix"
```

Comment:
Shouldn't this be using standard?

Reply:
Yeah! In other repos in @orbitdb we're using or switching to  http://standardjs.com, instead of a custom eslint interface. See, for instance, https://github.com/orbitdb/orbit-db-identity-provider/blob/master/package.json. We can make this another issue, too.

Reply:
Thanks, I'll check into it and make changes accordingly.

Reply:
We discussed this with Jani and decided to make this into a separate issue/PR.